extensions/wikia/Email/Controller/PasswordResetLinkController.class.php

```diff
@@ -51,6 +53,25 @@ public function getSubject() {
 	 * @template passwordResetLink
 	 */
 	public function body() {
+		// PLATFORM-2762 - we need to log information about reset tokens
+		// this action should be triggered by anonymous user, so current user will most likely contain ip. as this
+		// will make the CheckUser logs harder to browse, we override currentUser with targetUser
```

Comment:
With the current reset flow, I believe current user is the IP. Setting it to the account being reset is misleading as it suggests that user performed the action when it might have been someone else.